site/en/docs/webstore/cws-dashboard-listing/index.md

```diff
@@ -54,9 +71,31 @@ official URL pulldown", width="800", height="184" %}
 
 The pull-down for this field contains only those sites that have been verified as belonging to you.
 To add a verified site that you own, click on **Add a new site**. This opens the Google Search
-Console, where you can add and verify a site. See [Verify your site ownership][verify-owner] for
+Console, where you can add and verify a site. See [Verify your site ownership][4] for
 more details.
 
-[cws-branding]: /docs/webstore/branding
-[cws-images]: /docs/webstore/images
-[verify-owner]: https://support.google.com/webmasters/answer/9008080
+### Adding a homepage URL {: #home-url }
+
+If you want to explain how your extension works in more detail, you can provide a direct link to your extension's website in the **Homepage URL**. It will be located under the Additional information of your item.
```

Comment:
Suggestion:

To explain how your extension works in more detail, you can provide a direct link to your extension's website in the **Homepage URL**. This link appears under the Additional information of your item.

Rationale:
* It always bugs me when something is "designed to" instead of "does". It doesn't matter about the design intention (usually) but only what the behavior is. Similarly here, it doesn't matter if you want to explain, just this is how you explain. See what I mean? Pet peeve.
* Avoid "will" almost every time ("it will"), and passive voice ("be located"). (https://developers.google.com/style/tense , https://developers.google.com/style/voice)

Reply:
Gotcha! To improve in the future, I will read the attached style docs.